exercises/concept/lasagna-master/.meta/exemplar.go

```diff
@@ -0,0 +1,35 @@
+package lasagna
+
+// Determines whether the lasagna is done based on the
+func cookingStatus(time int) string {
```

Comment:
Should these functions start with a capital letter? I've seen that used in other exercises.